File: 0242-valid-anagram/0242-valid-anagram.go
func isAnagram(s string, t string) bool {
    chars := make([]int, 26)

    for _, v := range s {
        chars[v - 'a']++;
    }

    for _, v := range t {
        chars[v - 'a']--;
    }

    for _, v := range chars {
        if v != 0 {
            return false
        }
    }

    return true
}

File: 0242-valid-anagram/0242-valid-anagram.rs
impl Solution {
    pub fn is_anagram(s: String, t: String) -> bool {
        let mut chars: [i32; 26] = [0; 26];

        for v in s.chars() {
            let idx = v as u32 - 'a' as u32;
            chars[idx as usize] += 1;
        }

        for v in t.chars() {
            let idx = v as u32 - 'a' as u32;
            chars[idx as usize] -= 1;
        }

        for v in chars {
            if (v != 0) {
                return false;
            }
        }

        return true;
    }
}